xbmc/cores/VideoPlayer/VideoRenderers/DebugRenderer.h

```diff
@@ -12,14 +12,21 @@
 
 #include <string>
 
+#define NUM_ELE 6
```

Comment:
"ELE" is rather ambiguous. Can you document the constant?

Reply:
Changed to MAX_LINES